docs/environment-variables.md

```diff
@@ -1,28 +1,53 @@
 # Environment Variables
```

Comment:
I assume this got sorted and nothing else.

Reply:
Mostly, yes. Sorry, forgot to mention that. I've updated the description to note this, too.

I expanded the RULES_PYTHON_EXTRACT_ROOT docs a bit, but the same in spririt.